pubsub/azuresb/azuresb.go

```diff
@@ -342,29 +345,41 @@ func openSubscription(ctx context.Context, sbNs *servicebus.Namespace, sbTop *se
 	if opts == nil {
 		opts = &SubscriptionOptions{}
 	}
+	// One-time initialization of Link to AMQP server, lazily initiated in
+	// ReceiveBatch so that we delay "invalid topic/subscription" errors until
```

Comment:
Why couldn't you just call it here, but save the error and report it in ReceiveBatch?

Reply:
Done.